meta-freescale-mbl/recipes-bsp/u-boot/u-boot-fslc_%.bbappend

```diff
@@ -8,18 +8,82 @@ MBL_UBOOT_VERSION_imx6ul-pico-mbl = "v2019.10-rc1"
 
 inherit mbl-uboot-sign
 
-SRCREV = "c0c4ee5fce01ec0818c4f27ce029d9b16c8849ad"
-SRCREV_imx6ul-pico-mbl = "ee3b9f43dfdde29ba939342bbdc22a0549015979"
+SRCREV = "a1588ac8228881f9fe65539fa8e31f0ee3556864"
+SRCREV_imx6ul-pico-mbl = "dcf722ece6aad0c1512a26cdefc2f74a192fa9d2"
 
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
-SRC_URI = "git://git.linaro.org/landing-teams/working/mbl/u-boot.git;protocol=https;nobranch=1 \
-           ${@bb.utils.contains_any('PACKAGECONFIG','noconsole silent',' file://0002-set-silent-envs.patch','',d)} \
+SRC_URI = "git://git.denx.de/u-boot.git;protocol=https;nobranch=1 \
```

Comment:
For future reference, its unnecessary to continue this line across multiple lines as the line fits on one line. As we're under time pressure to get this accepted I'm not intending to gate this PR on this change.